session.go

```diff
@@ -72,7 +72,7 @@ func (s Session) IsValid(r *http.Request) bool {
 		browser = a != nil && s.Agent.Browser == a.Name
 	}
 
-	return ip && os && browser
+	return ip && os && browser && s.ExpiresAt.After(time.Now())
```

Comment:
It would be better to move this check to `if !ok || {expiration check} {...}` in the `wrap()` method. The purpose of `IsValid()` is to check if the request's data is valid/matches what is in the active session (this can also be disabled/enabled and expiration timestamps should always be checked). 

The place where expiration timestamps should be checked: https://github.com/swithek/sessionup/blob/81210122cd330a184f860db2016b8ffcf6bef894/manager.go#L288-L291